services/dy-static-file-server/src/dy_static_file_server/.gitkeep

```diff
@@ -0,0 +1 @@
+add source code here
```

Comment:
No need for the .gitkeep files

Reply:
I'd like to keep it, in case later on we modify that service